octobot_trading/modes/abstract_trading_mode.py

```diff
@@ -118,6 +143,23 @@ def get_supported_exchange_types(cls) -> list:
             enums.ExchangeTypes.SPOT
         ]
 
+    def get_mode_producer_classes(self) -> list:
+        return self.MODE_PRODUCER_CLASSES
+
+    def get_mode_consumer_classes(self) -> list:
+        return self.MODE_CONSUMER_CLASSES
+
+    def should_emit_trading_signals_user_input(self, inputs: dict):
+        if self.user_input(
+            common_constants.CONFIG_EMIT_TRADING_SIGNALS, common_enums.UserInputTypes.BOOLEAN, False, inputs,
+            title="Emit trading signals on Astrolab for people to follow.", order=998
```

Comment:
What is `order=998`?

Reply:
order is the order of appearance in the config form. We could change it to `constant.LAST_INPUT-1` for example, do you think it would be better ?

Reply:
Yes it would be better😉